core/embed/rust/src/ui/layout_eckhart/ui_firmware.rs

```
@@ -468,6 +471,9 @@ impl FirmwareUI for UIEckhart {
         let header = if info {
             Header::new(title)
                 .with_right_button(Button::with_icon(theme::ICON_INFO), HeaderMsg::Menu)
+        } else if external_menu {
```

nit: maybe raise an exception if both `info` and `external_menu` are set?